.changeset/funny-apples-wash.md

```diff
@@ -0,0 +1,17 @@
+---
+"@fluidframework/container-loader": major
+"@fluidframework/runtime-utils": major
+"@fluidframework/aqueduct": major
```

Comment:
I hesitated reviewing this bit. IIRC these are being set to `major` because we're in `next`, but these changes aren't technically breaking even if they're being done in next... should this still say `major`? @tylerbutler for thoughts, since you probably have the most context around the rationale here.

Alternate question: we should be able to do this whole change in `main`, right? Why not do it there?

Reply:
This PR is following making entryPoint required https://github.com/microsoft/FluidFramework/pull/17040. So, we either do this PR now in `next` or wait for it to be released and then do the change in `main`. I'd rather do the change now and give consumers more notice if they don't decide to pick up minor versions.

Reply:
Still should probably change it to minor, right?  @tylerbutler?